tests/bluetooth/tester/src/btp/btp_gap.h

```diff
@@ -599,3 +600,32 @@ int tester_gap_padv_stop(struct bt_le_ext_adv *ext_adv);
 int tester_gap_padv_create_sync(struct bt_le_per_adv_sync_param *create_params);
 int tester_gap_padv_stop_sync(void);
 #endif /* defined(CONFIG_BT_EXT_ADV) */
+
+#if defined(CONFIG_BT_EAD)
+#define BTP_GAP_EAD_SET_KEY_MATERIAL 0x31
+#define BTP_GAP_EAD_CREATE_ADV_DATA  0x32
+#define BTP_GAP_EAD_DECRYPT_ADV_DATA 0x33
+
+#define BTP_GAP_EAD_KEY_SIZE 16
+#define BTP_GAP_EAD_IV_SIZE  8
```

Comment:
typically we name flags with command prefix, ie here it would be 
BTP_GAP_EAD_SET_KEY_MATERIAL_KEY_SIZE
BTP_GAP_EAD_SET_KEY_MATERIAL_IV_SIZE

same for structure names (cmd and rp)
struct btp_gap_ead_encrypt_adv_data_cmd
struct btp_gap_ead_decrypt_adv_data_cmd